docs/dev_install.rst

```diff
@@ -34,7 +34,7 @@ Installing Depedencies
     mkdir build
     cd build
     cmake ..
-    make -j 8
+    make -j$(nproc)
```

Comment:
note `nproc` is not avaialble on macOS. Might want to try something like `$(nproc 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || echo 8)` to make it portable and have a default fallback?

Reply:
Great suggestion to apply!
I do not have a mac machine but ```2>/dev/nul``` ensures no error will be inputted, and ```|| echo 8``` does ensure the fallback. 
The safest way is ``` echo 1```,nonetheless, if 8 has been fine so far I assume the developers who indeed visit this page will find it easy to determine a proper job number.